src/main/java/seedu/address/model/FoodBook.java

```diff
@@ -0,0 +1,121 @@
+package seedu.address.model;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import javafx.collections.ObservableList;
+import seedu.address.model.food.Food;
+import seedu.address.model.food.UniqueFoodList;
+
+
+/**
+ * Wraps all data at the address-book level
+ * Duplicates are not allowed (by .isSameFood comparison)
+ */
+public class FoodBook implements ReadOnlyFoodBook {
+
+    private final UniqueFoodList foods;
```

Comment:
can just initialise with the new UniqueFoodList() here instead of within the block. Not sure if it will solve the error but worth trying. As long as not within the constructor as they are scared of duplication.

Reply:
Resolved.